app/domain/errors.rb

```diff
@@ -166,6 +166,11 @@ module Jwt
         code: "CONJ00051E"
       )
 
+      RequestBodyIsNotJWTToken = ::Util::TrackableErrorClass.new(
+        msg: "The request body is not a JWT token",
```

Comment:
@shulifink please review

Reply:
@hilagross ^

Reply:
What is a "reques body" - it what does it mean that it's not a JWT token? 
Do you mean that the request doesn;t include/contain a JWT token?